cli/jira_config_gen/__init__.py

```diff
@@ -30,14 +30,19 @@
     "--token_path",
     help="Path to the Jira API token",
     required=True,
-    type=click.STRING,
+    type=click.Path(exists=True),
 )
 @click.option(
     "--output_file",
     help="Where the rendered config will be stored",
-    required=True,
     default="/tmp/jira.config",
-    type=click.STRING,
+    type=click.Path(),
```

Comment:
should have `exists=True` right?

Reply:
@myakove No because this is the output file. It wouldn't exist until after this command is run.